device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/CalendarService.kt

```diff
@@ -112,27 +146,32 @@ public class CalendarService : PluginRegistry.RequestPermissionsResultListener {
             try {
                 if (cursor?.moveToFirst() ?: false) {
                     val calendar = parseCalendar(cursor);
-                    return calendar;
+                    if (isInternalCall) {
+                        return calendar;
+                    } else {
+                        finishWithSuccess(_gson?.toJson(calendar));
+                    }
                 } else {
-
+                    if (!isInternalCall) {
+                        finishWithError(CALENDAR_RETRIEVAL_FAILURE, "Couldn't retrieve the Calendar with ID ${calendarId}");
```

Comment:
can the "Couldn't retrieve..." string be made a constant to do or is it not possible to do?

Reply:
I don't know how to make it constant/static with the formatting in it